doc/sphinx-guides/source/developers/remote-users.rst

```diff
@@ -55,15 +55,32 @@ Now load the configuration defined in ``oidc-keycloak-auth-provider.json`` into
 
 You should see the new provider, called "OIDC-Keycloak", under "Other options" on the Log In page.
 
-You should be able to log into Keycloak with the following credentials:
-
-- username: kcuser
-- password: kcpassword
+You should be able to log into Keycloak with the one of the following credentials:
+
+.. list-table::
+
+  * - Username
+    - Password
+    - Group
+  * - admin
+    - admin
+    - admins
+  * - curator
+    - curator
+    - curators
+  * - user
```

Comment:
If you feel this is a large problem, I'd favor generic names (not user1...n, because implicit semantics).

On the other hand, this kind of thing is hard to change later on because people will start to rely on it. Having semantics in the name makes things easier to read when getting familiar with tests etc, as you don't have to do the mapping in your head.

Reply:
Ok, let's stick with admin, curator, etc.

By the way, I finally got around to reading https://martinfowler.com/bliki/ObjectMother.html from https://github.com/IQSS/dataverse-frontend/pull/165#issuecomment-1715344841 and it's interesting:

> Maybe 'John', an employee who just got hired last week; 'Heather' and employee who's been around for a decade....
> The canned objects that the Object Mother produces become familiar to the team, often invading even discussions with the users. In this way they resemble the notion of personas - although they aren't always people. They could be insurance policies, supply contracts, whatever data a test framework needs. Using similar data on multiple tests helps people be familiar with the examples you're using.
> 
> These canned objects often aren't just right for a particular test, but often can be made right with some additional setup. "Let's take John and make him go off sick two months ago." Occasionally you'll need to add a new canned object to the mother, but try to tweak an existing one if you can - that way the reader of the test will understand quicker if they are familiar with the existing canned objects.

Reply:
Great! I'll resolve this now :smile: